src/controllers/idea.controller/getIdea/recommender/index.js

```diff
@@ -0,0 +1,84 @@
+const { TfIdf } = require('natural');
+const { createUserProfile } = require('../../../../utils/recommendations/helper/userProfile');
+const { createIdeaProfile } = require('./ideaProfile');
+const {
+  calculateCosineSimilarity,
+} = require('../../../../utils/recommendations/helper/vectorBasedMapping');
+
+const recommend = async (user, ideas) => {
```

Comment:
rename the user argument to cognitoSub since that is what is being sent by the controller